src/ui/Base/RegisteredNamedMethods.ts

```diff
@@ -447,6 +453,54 @@ Initialization.registerNamedMethod('configureResourceRoot', (path: string) => {
   configureResourceRoot(path);
 });
 
+/**
+ * Re-enables all [Analytics]{@link Analytics} components if they were previously disabled.
+ * @param searchRoot What ancestor or CoveoAnalytics element to search from in the DOM.
```

Comment:
Suggestion:

> The element to scan for an `Analytics` component. This can be an element onto which a component instance is bound (e.g., `document.querySelector(".CoveoAnalytics")`, or an ancestor of such an element (e.g., `document.getElementById("search")`.